### PR TITLE
Used POST to add a comment to comments

### PR DESCRIPTION
**Pull request by zahraakarim**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -172,3 +172,104 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+      .post("/api/articles/2/comments")
+      .send({
+        username: "zahraa",
+        body: "I'm adding a new comment, let's goooo!",
+      })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Path not found");
+      });
+  });
```
Comment (roisinoreilly) [inline __tests__/app.test.js:274]: really excellent error handling, well done 🌟

File: app.js [hunk 2]
```diff
@@ -20,16 +21,26 @@ app.get("/api/articles", getAllArticles);
 
 app.get("/api/articles/:article_id/comments", getAllArticleComments);
 
+app.post("/api/articles/:article_id/comments", postComment);
+
 app.all("*", (_, res) => {
   res.status(404).send({ msg: "Path not found" });
 });
 
 app.use((err, req, res, next) => {
```
Comment (roisinoreilly) [inline app.js:30]: at some point I would think about extracting your error handling middleware into a separate file to keep your app neat :)

File: news.model.js [hunk 3]
```diff
@@ -40,3 +38,14 @@ exports.selectAllArticleComments = (article_id) => {
       return response.rows;
     });
 };
+
+exports.addComment = (article_id, username, body) => {
+  return db
+    .query(
+      "INSERT INTO comments (article_id, author, body) VALUES ($1 , $2, $3) RETURNING *;",
```
Comment (roisinoreilly) [inline news.model.js:45]: really nicely done

**Review by roisinoreilly — APPROVED** [review]
really great work with your testing, keep it up!! PR approved :)

File: app.js (hunk 2 above)
Comment (zahraakarim) [reply app.js:30]: For sure will, was just leaving any housekeeping till then end!

**Pull request merged**